module Async
  class CancelledError < Exception; end
end
